io/disk/tiobench.py.data/tiobench.yaml

```diff
@@ -0,0 +1,26 @@
+setup: 
+    location: !mux
+        default:
+            target: '/mnt'
+
+    block: !mux
+        first:
+            blocks: 4096
+        second:
+            blocks: 8192
+
+    thread: !mux
+        default:
+            threads: 10
+
+    blocksize: !mux
+        default: 
+            size: 1024
+     
+    runs: !mux
+        default: 
+            numruns: 2
+  
+    iteration: !mux
+        default:
+            iterations: 5
```

Comment:
Btw you can verify it by `avocado multiplex io/disk/tiobench.py.data/tiobench.yaml` (use `-h` to see options).